tests/regression/action/test_action_classification.py

```diff
@@ -0,0 +1,134 @@
+"""Tests for Action Classification with OTX CLI."""
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+import json
+import os
+from pathlib import Path
+from timeit import default_timer as timer
+
+import pytest
+
+from otx.cli.registry import Registry
+from tests.regression.regression_test_helpers import (
+    REGRESSION_TEST_EPOCHS,
+    TIME_LOG,
+    get_result_dict,
+    load_regression_configuration,
+)
+from tests.test_suite.e2e_test_system import e2e_pytest_component
+from tests.test_suite.run_test_command import (
+    otx_eval_compare,
+    otx_eval_openvino_testing,
+    otx_export_testing,
+    otx_train_testing,
+    pot_eval_testing,
+    pot_optimize_testing,
+)
+
+# Configurations for regression test.
+TASK_TYPE = "action_classification"
+TRAIN_TYPE = "supervised"
+LABEL_TYPE = "multi_class"
+
+otx_dir = os.getcwd()
+templates = Registry("otx/algorithms/action").filter(task_type=TASK_TYPE.upper()).templates
+templates_ids = [template.model_template_id for template in templates]
+
+
+result_dict = get_result_dict(TASK_TYPE)
+result_dir = f"/tmp/regression_test_results/{TASK_TYPE}"
```

Comment:
It's simple question, is there a logic to delete result_dir?

Reply:
As far as I know, crontab cleans the /tmp folder regularly.